src/averageAges.js

```diff
@@ -15,47 +15,62 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let males = people.filter(({ sex }) => sex === 'm');
+
+  if (century) {
+    males = males.filter(male => Math.ceil(male.died / 100) === century);
+  }
+
```

Comment:
in the checklist point it is mentioned that 
` replace `if ()` statement with &&, || or ?:`
so remove all if conditions and replace with `turnery` condition

Reply:
let's do that – you can put `century` validation into `people.filter(...)`